examples/reverse-proxy/MNIST_using_Reverse_Proxy.md

```diff
@@ -0,0 +1,66 @@
+# MNIST using Reverse Proxy
+
+This is single host MNIST example using reverse proxy service parameters with 1 SN node, 1 SWOP node, 1 SWCI node and 2 SL nodes.
+
+The following image illustrates a cluster setup for the Reverse Proxy with MNIST example:
+
+![](GUID-D9C8F73B-BAFB-41E2-9B70-02C628836B96-high.png)
+
+1.  This example uses a SN node. This node is named as **sn-1** and is the Sentinel Node. **sn-1** runs on host 172.1.1.1.
+
+2.  SL and ML nodes are automatically spawned by SWOP node during training.
+
+3.  This example uses a SWOP node that connects to SN node. The name of the docker container representing this SWOP node is **swop-1**. **swop-1** runs on host 172.1.1.1.
```

Comment:
[README.md](https://github.hpe.com/hpe/swarm-learning-docs/tree/master/examples/reverse-proxy/mnist#readme)

Compare texts once and rephrase them.